frontend/integration-tests/views/devconsole-view/template.view.ts

```diff
@@ -0,0 +1,64 @@
+// import * as _ from 'lodash';
+import { $, $$, browser, by, ExpectedConditions as until } from 'protractor';
+
+import { appHost, testName, waitForNone } from '../../protractor.conf';
+import { resourceTitle } from '../../views/crud.view';
+
+
+export const reloadBtn = $('#reload-object');
+
+//element finder with class name
+export const untilLoadingBoxLoaded = until.presenceOf($('.loading-box__loaded'));
+export const untilNoLoadersPresent = waitForNone($$('.co-m-loader'));
+
+export const isLoaded = () => browser.wait(until.and(untilNoLoadersPresent, untilLoadingBoxLoaded)).then(() => browser.sleep(1000));
+
+export const resourceRows = $$('[data-test-rows="resource-row"]');
+// export const resourceRowNamesAndNs = $$('.co-m-resource-icon + a');
+
+export const rowForName = (name: string) => resourceRows.filter((row) => row.$$('.co-m-resource-icon + a').first().getText().then(text => text === name)).first();
```

Comment:
This is definitely *not* a best practice and leads to `StaleElementReferenceError` during tests. We shouldn't encourage this pattern.